### PR TITLE
Updating Auth0 Migration guide

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Making minor edits to the workflows based on final designs

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/cloud/manage-access/auth0-migration.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Once you have opted to begin the migration process, the following steps will var
 
 :::warning Login {slug}
 
-The login slug shouldn't have an underscore. If it does, you need to change it on the **Account Settings** page. After changing the slug, the admins must share it with dbt Cloud users, as the login URL will differ.
+The login slug shouldn't have an underscore. If it does, you need to change it before migrating on the **Account Settings** page or during the migration on the final screen before enabling (see screenshots for your respective setup). After changing the slug, the admins must share it with dbt Cloud users, as the login URL will differ.
```
Comment (runleonarun) [inline website/docs/docs/cloud/manage-access/auth0-migration.md]: Is the action here to remove the underscore from all logins? Maybe this message can be condensed:
```suggestion
Make sure to remove underscores (if they exist) from login slugs: 
* before migrating on the **Account Settings** page, or 
* while migrating (but before enabling) as show in the Migrate authentication screenshots for your respective setup. 
After changing the slug, admins must share the new login URL with their dbt Cloud users.
```

**Review by runleonarun — COMMENTED** [review]
Hey @matthewshaver! Just a minor suggestion to rewrite the warning by starting with the action they need to do.

I also see 4 warning notes on this page and that feels like a lot and also like they might be glossed over because there are so many.  Can any of these warnings be written into steps?

**Comment by matthewshaver** [comment]
> Hey @matthewshaver! Just a minor suggestion to rewrite the warning by starting with the action they need to do.
> 
> I also see 4 warning notes on this page and that feels like a lot and also like they might be glossed over because there are so many. Can any of these warnings be written into steps?

@runleonarun I would agree in any other scenario but I made the choice to do it like this because people aren't going to read this entire doc, they're going to skip to the section that's pertinent to their existing configuration. Does that make sense?

**Review by runleonarun — APPROVED** [review]
LGTM!

**Pull request merged**